### PR TITLE
Flatpak for org.gnode.NixView

### PR DESCRIPTION
**Pull request by gicmo**
[NixView](http://bendalab.github.io/NixView/) is a data viewer for the [NIX file-format](https://github.com/G-Node/nix/).

### CODE REVIEW

File: org.gnode.NixView.desktop [hunk 1]
```diff
@@ -0,0 +1,7 @@
+[Desktop Entry]
+Type=Application
+Name=NixView
+Comment=Viewer for nix datafiles
+Icon=org.g-node.NixView
```
Comment (cosimoc) [inline org.gnode.NixView.desktop]: Should this not be `org.gnode.NixView` instead?

File: org.gnode.NixView.json [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+		    "path": "org.gnode.NixView.appdata.xml"
+		},
+		{
+		    "type": "file",
+		    "path": "org.gnode.NixView.desktop"
+		},
+		{
+		    "type": "file",
+		    "path": "org.gnode.NixView.png"
+		}
```
Comment (cosimoc) [inline org.gnode.NixView.json]: How are these files installed?

**Comment by gicmo** [comment]
Thanks a lot @cosimoc. I pushed some fixes.

**Comment by TingPing** [comment]
Test build: https://flathub.org/builds/#/builders/1/builds/454

**Comment by TingPing** [comment]
Test build: https://flathub.org/builds/#/builders/1/builds/456

**Comment by cosimoc** [comment]
Seemed to have failed building on ARM...

**Comment by gicmo** [comment]
I just pushed a patch that should fix the ARM build errors. Patch taken from G-Node/nix#700

**Comment by TingPing** [comment]
Test build: https://flathub.org/builds/#/builders/1/builds/461

File: org.gnode.NixView.json [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+    "runtime-version": "1.6",
+    "sdk": "org.freedesktop.Sdk",
+
+    "finish-args": [
+	"--filesystem=home:ro",
+	"--share=ipc", "--socket=x11",
+        "--socket=pulseaudio",
+        "--share=network",
+	"--device=dri",
+	"--socket=wayland"
```
Comment (TingPing) [inline org.gnode.NixView.json]: We tend to disable Wayland on Qt applications for now.

File: org.gnode.NixView.json [hunk 4]
```diff
@@ -0,0 +1,115 @@
+{
+    "app-id": "org.gnode.NixView",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "1.6",
+    "sdk": "org.freedesktop.Sdk",
+
+    "finish-args": [
+	"--filesystem=home:ro",
+	"--share=ipc", "--socket=x11",
+        "--socket=pulseaudio",
```
Comment (TingPing) [inline org.gnode.NixView.json]: Whitespace nit-pick.

File: org.gnode.NixView.json [hunk 5]
```diff
@@ -0,0 +1,115 @@
...
+		    "path": "org.gnode.NixView.desktop"
+		},
+		{
+		    "type": "file",
+		    "path": "org.gnode.NixView.png"
+		}
+	    ],
+	    "post-install": [
+		"install -Dm644 org.gnode.NixView.desktop /app/share/applications/org.gnode.NixView.desktop",
+                "install -Dm644 org.gnode.NixView.png /app/share/icons/hicolor/256x256/apps/org.gnode.NixView.png",
```
Comment (TingPing) [inline org.gnode.NixView.json]: Whitespace nitpick.

**Comment by gicmo** [comment]
Fixed the whitespace issues, added the release tag.

**Comment by TingPing** [comment]
Repository has been created: https://github.com/flathub/org.gnode.NixView

**Pull request closed without merging**